modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/instance/lifecycle/AddObjectDefinitionPortalInstanceLifecycleListener.java

```diff
@@ -0,0 +1,114 @@
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.model.Company;
+import com.liferay.portal.kernel.model.Release;
+import com.liferay.portal.kernel.model.Role;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.model.role.RoleConstants;
+import com.liferay.portal.kernel.security.auth.PrincipalThreadLocal;
+import com.liferay.portal.kernel.security.permission.PermissionCheckerFactory;
+import com.liferay.portal.kernel.security.permission.PermissionThreadLocal;
+import com.liferay.portal.kernel.service.RoleLocalService;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Paulo Albuquerque
+ */
+@Component(service = PortalInstanceLifecycleListener.class)
+public class AddObjectDefinitionPortalInstanceLifecycleListener
+	extends BasePortalInstanceLifecycleListener {
+
+	@Override
+	public void portalInstanceRegistered(Company company) throws Exception {
+		if (FeatureFlagManagerUtil.isEnabled("LPS-167253")) {
+			_importJSONObjectDefinition(company);
+		}
+	}
+
+	private void _importJSONObjectDefinition(Company company) throws Exception {
+		Role role = _roleLocalService.getRole(
+			company.getCompanyId(), RoleConstants.ADMINISTRATOR);
+
+		long[] currentUserId = _roleLocalService.getUserPrimaryKeys(
+			role.getRoleId());
+
+		User currentUser = _userLocalService.getUser(currentUserId[0]);
+
+		PermissionThreadLocal.setPermissionChecker(
+			_defaultPermissionCheckerFactory.create(currentUser));
+
+		PrincipalThreadLocal.setName(currentUser.getUserId());
+
+		ObjectDefinitionResource.Builder objectDefinitionResourceBuilder =
+			_objectDefinitionResourceFactory.create();
+
+		ObjectDefinitionResource objectDefinitionResource =
+			objectDefinitionResourceBuilder.user(
+				currentUser
+			).build();
+
+		JSONObject objectDefinitionJSONObject = _jsonFactory.createJSONObject(
+			StringUtil.read(getClass(), "dependencies/bookmarks.json"));
+
+		ObjectDefinition objectDefinition = ObjectDefinition.toDTO(
+			objectDefinitionJSONObject.toString());
+
+		objectDefinition =
+			objectDefinitionResource.putObjectDefinitionByExternalReferenceCode(
+				objectDefinition.getExternalReferenceCode(), objectDefinition);
+
+		Status status = objectDefinition.getStatus();
+
+		if (status.getCode() != 0) {
+			objectDefinitionResource.postObjectDefinitionPublish(
+				objectDefinition.getId());
+		}
+	}
+
+	@Reference
+	private PermissionCheckerFactory _defaultPermissionCheckerFactory;
+
+	@Reference
+	private JSONFactory _jsonFactory;
+
+	@Reference
+	private ObjectDefinitionResource.Factory _objectDefinitionResourceFactory;
+
+	@Reference(
+		target = "(&(release.bundle.symbolic.name=com.liferay.object.service)(release.schema.version>=1.0.0))"
+	)
+	private Release _release;
```

Comment:
Do we really need this? If the answer is yes, we need to fix the target. This Reference is not correct, I change the `release.bundle.symbolic.name` to `com.liferay.object.admin.rest.impl`, but it doesn't works. The Bookmarks definition is not created in the startup process.
If the answer is no, I can remove this Reference and it will works.

Reply:
cc: @gabrielwas, @marcelabc

Reply:
After sync the branch with master, It is working

Reply:
I believe that we need to change the `release.schema.version>=1.0.0` to `release.schema.version>=4.1.0` because it's the version that we are adding the modifiable column to object definition. You can check this [thread](https://liferay.slack.com/archives/C03LW8NUK/p1678911023691929) for more details.

Reply:
cc: @mateussgn

Reply:
Hey guys, I made some adjustments to @p-albuquerque [pull request](https://github.com/liferay-objects/liferay-portal/pull/1759). It's in a [pull request](https://github.com/guilhermedcamacho/liferay-portal/pull/131) on my repo. 
It's working but we have a limitation regarding to have layouts or views in Json file, because we need permissions for that. I can explain better my findings tomorrow.